client/dashboard/sites/settings-defensive-mode/index.tsx

```diff
@@ -217,10 +218,10 @@ export default function DefensiveModeSettings( { siteSlug }: { siteSlug: string
 					title={ __( 'Defensive mode' ) }
 					description={ createInterpolateElement(
 						__(
-							'Extra protection against spam bots and attacks. Visitors will see a quick loading page while we run additional security checks. <learnMoreLink />'
+							'Extra protection against spam bots and attacks. Visitors will see a quick loading page while we run additional security checks. <link>Learn more</link>.'
```

Comment:
Should all these `Learn more` links (in page subtitles) have period at the end, @matt-west? I think we have some inconsistency; it's better to codify it / make a convention.

Reply:
Let's remove the period when it’s just Learn more ↗️